README.md

```diff
@@ -388,3 +397,24 @@ Alternatively:
   RedisEnterpriseImageSpec:
     versionTag: redislabs/redis:6.0.6-39
 ```
+
+## Supported K8S Distributions
+Each release of the Redis Enterprise Operator deployment is thoroughly tested against a set of Kubernetes distributions. The table below lists these, along with the current release's support status. "Supported", as well as "deprecated" support status indicates the current release has been tested in this environment and supported by RedisLabs. "Deprecated" also indicates that support will be dropped in a coming future release. "No longer supported" indicates that support has been dropped for this distribution. Any distribution that isn't explicitly listed is not supported for production workloads by RedisLabs. 
+| Distribution      | Support Status      |
+|-------------------|---------------------|
+| Openshift 3.11    | deprecated          |
```

Comment:
@yuvallevy2 did we say we're deprecating now already, although it may be further int he future?

Reply:
i am surprised i have no idea where this came from

Reply:
Roey added it, https://github.com/RedisLabs/redis-enterprise-operator/pull/715, you approved